packages/cli/src/commands/init/templateName.ts

```diff
@@ -77,6 +78,19 @@ function handleVersionedPackage(versionedPackage: string) {
   };
 }
 
+function handleGitHubRepo(templateName: string) {
+  if (['github', '@'].some(str => templateName.includes(str))) {
```

Comment:
I don't think our unit test handles this `if` statement, right? Can we have a case for it too?

Reply:
I think we should remove this and also improve the regex to not trigger on URLs, otherwise, this will break and affect those who init with a URL or git.

(author probably tested with Git URL only hence `github` and `@` here)